pkg/parca/parca_test.go

```diff
@@ -34,7 +34,7 @@ func benchmarkSetup(ctx context.Context, b *testing.B) (pb.ProfileStoreServiceCl
 	done := make(chan struct{})
 	go func() {
 		defer close(done)
-		err := Run(ctx, logger, reg, &Flags{ConfigPath: "testdata/parca.yaml", Port: ":9090"})
+		err := Run(ctx, logger, reg, &Flags{ConfigPath: "testdata/parca.yaml", Port: ":9090"}, nil)
```

Comment:
We can't pass `nil` here. Let's just do `test-version` or something like that.

Reply:
:heavy_check_mark:  TY